controllers/auth.js

```diff
@@ -49,10 +49,10 @@ const githubAuthCallback = (req, res, next) => {
         logger.error(err);
         return res.boom.unauthorized("User cannot be authenticated");
       }
-
       userData = {
         github_id: user.username,
         github_display_name: user.displayName,
+        github_created_at: user._json.created_at,
```

Comment:
Can we store the timestamp in Unix timestamp for consistency?

Reply:
OK, I will change it

Reply:
done